### ISSUE TITLE
Exception when running the executable python3.10 in cpython.jar

### ISSUE THREAD

**Issue by huazai023** [bytedeco/javacpp-presets#1363]
Environment: Android real machine + termux
Operation: An exception is thrown when using the cppbuild.sh script to execute the command python-path/python3.10 -m pip install --target=install-path/python numpy: ./python3.10: cannot execute: required file not found
Version: Version: cpython all linux-arm64 versions

**Comment by saudet** [comment]
We probably need to fix a couple of things to get Java SE working with Android

**Comment by huazai023** [comment]
thanks

**Comment by huazai023** [comment]
Can you provide cpython's Android platform compilation script?

**Comment by huazai023** [comment]
Yes, the way I compiled it before was wrong, so the question I asked did not accurately express what I meant

**Comment by saudet** [comment]
If you're asking if you could hire me to work on this, yes, that's an option.

**Comment by huazai023** [comment]
I have solved the problem and can compile a usable android version

**Comment by huazai023** [comment]
After I verify it, can I submit a pull request and merge it in?

**Comment by huazai023** [comment]
would a cup of coffee be ok ^_^？
